docs/changelog/4.3.rst

```diff
@@ -0,0 +1,113 @@
+Changelogs for 4.3.x
+====================
+
+.. changelog::
+  :version: 4.3.0-alpha1
+  :released: 10th of December 2019
+
+  Compared to the last release candidate, one more bug has been fixed.
+
+  The LMDB backend is incomplete in this version. Slaving zones works, loading zones with pdnsutil works, but more fine grained edits (using edit-zone, or the REST API) fail. We hope to fix this soon in a 4.2.x release.
```

Comment:
pretty sure this isn't the case anymore?

Reply:
lazy copy, will fix